src/ATA.Bluebook/ATA.Bluebook.Web/Views/Shared/_DevExtremeLayout.cshtml

```diff
@@ -1,3 +1,4 @@
+@inject IHttpContextAccessor Accessor
```

Comment:
Move it to _import.cshtml and rename the variable name with "_accessor"

Reply:
Done